integration/testrunner/config.go

```diff
@@ -75,6 +66,27 @@ type TestCaseConfig struct {
 	TalkerClient talkerpb.TalkerClient
 }
```

Comment:
We should parse the key early on in `TestMain` and store _one_ copy of it in `TestCaseConfig`. The current logic requires reparsing the token every time we need justification.

Reply:
Done